Samba-build/Samba-tests/src/test/java/io/avalia/samba/AfoxeTest.java

```diff
@@ -0,0 +1,37 @@
+package io.avalia.samba;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class AfoxeTest {
+
+	  @Test
+	  public void thereShouldBeAnIInstrumentInterfaceAndAAfoxeClass() {
+	    IInstrument afoxe = new Afoxe();
+	    assertNotNull(afoxe);
+	  }
+	  
+	  @Test
+	  public void itShouldBePossibleToPlayAnInstrument() {
```

Comment:
Be more precise when you name methods: are you specifying the behaviour of an INSTRUMENT or of an AFOXE?